### PR TITLE
Sara/mkt 388 integrating semgrep in ci

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Comment by s-santillan** [comment]
Hi @pabloest. This is the very first doc that I submitted last month. I reworked it to fit all the recent changes to our CI functionality and trimmed all the stuff that was better explained elsewhere (replaced with links). It's more or less the same doc as in the draft, but updated and tightened.

File: docs/semgrep-ci/overview.md [hunk 1]
```diff
@@ -1,262 +1,155 @@
...
-:::
-
-**Note**: Semgrep CI is a subcommand of Semgrep CLI, so you can test the code and behavior of Semgrep CI on your machine by running `semgrep ci`.
-
-## Adding Semgrep CI to your repository
-
-Semgrep CI behaves like other static analysis and linting tools: it runs a set of user-configured rules and returns a non-zero exit code if there are findings, resulting in its job showing a ✅ or ❌.
-
-By default, Semgrep CI suppresses internal errors. If Semgrep in CI encounters an error, it logs the error to the console, sends a report to a crash-reporting server, and then lets CI continue. This default behavior can be overridden. See the [Exit codes](#exit-codes) section below and [Configuring blocking findings and errors](/semgrep-ci/configuration-reference.md/#configuring-blocking-findings-and-errors).
+This document aims to help users choose which method best meets their goals in integrating Semgrep in their CI environment.
```
Comment (pabloest) [inline docs/semgrep-ci/overview.md:19]:
```suggestion
This document helps users choose which method best meets their goals in integrating Semgrep in their CI environment.
```

File: docs/semgrep-ci/overview.md [hunk 2]
```diff
@@ -1,262 +1,155 @@
...
 
-Copy the relevant template for your CI provider from the sections below. Read through the comments in the template to adjust Semgrep CI scan settings, selecting pull and merge requests, full scans on your branch.
-
-Once Semgrep CI is running, explore the [Semgrep Registry](https://semgrep.dev/explore) to find and add more project-specific rules.
-
-See [Advanced Configuration](#advanced-configuration) for further customizations, such as ignoring files and tuning performance.
-
-### GitHub Actions
+After selecting a method, refer to the links in the [Next steps](#next-steps) section for documentation on how to set up or refine your Semgrep in CI job.
```
Comment (pabloest) [inline docs/semgrep-ci/overview.md:21]:
```suggestion
After selecting a method, refer to the links in the [Next steps](#next-steps) section for documentation on how to set up or refine your CI job.
```

File: docs/semgrep-ci/overview.md [hunk 3]
```diff
@@ -1,262 +1,155 @@
...
 
 :::info
-You can add Semgrep CI automatically to a GitHub repository by clicking "Set up" on the [Projects page](https://semgrep.dev/manage/projects) of Semgrep App. You'll be able to adjust pull request and  merge behavior before Semgrep App asks to commit a workflow file to your repository.
+When running in CI, Semgrep runs fully in your build environment. Your code is never sent anywhere.
```
Comment (pabloest) [inline docs/semgrep-ci/overview.md:24]:
```suggestion
When running in CI, Semgrep runs fully in the CI build environment. Your code is never sent anywhere.
```

maybe?

File: docs/semgrep-ci/overview.md [hunk 4]
```diff
@@ -1,262 +1,155 @@
...
+* Semgrep App collects [findings data](/docs/managing-findings/#semgrep-ci), which includes the line number of the code match, **not the code**. It is hashed using a one-way hashing function. Findings data is used to generate hyperlinks and support other Semgrep functions.
```
Comment (pabloest) [inline docs/semgrep-ci/overview.md:25]:
```suggestion
* Semgrep App collects [findings data](/docs/managing-findings/#semgrep-ci), which includes the line number of the code match, but **not the code**. It is hashed using a one-way hashing function. Findings data is used to generate line-specific hyperlinks to your source code management system and support other Semgrep functions.
```

File: docs/semgrep-ci/overview.md [hunk 5]
```diff
@@ -1,262 +1,155 @@
...
 :::
 
-To manually add Semgrep CI to GitHub Actions, add a `.github/workflows/semgrep.yml` file to your repository. Follow the [workflow syntax for GitHub Actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions).
+## Choosing a method
 
-See this [example GitHub Actions workflow configuration](../sample-ci-configs/#github-actions) for Semgrep CI.
+There are two methods to set up Semgrep in CI. 
 
-### GitLab CI/CD
+* To scale and automate your scans across many repositories, **use Semgrep App. **Semgrep App provides a centralized web application to manage findings and configure rulesets and notifications. This is the **recommended method** of integrating Semgrep in CI.
```
Comment (pabloest) [inline docs/semgrep-ci/overview.md:33]:
```suggestion
* To scale and automate your scans across many repositories, **use Semgrep App**. Semgrep App provides a centralized web application to manage findings and configure rulesets and notifications. This is the **recommended method** of integrating Semgrep in CI.
```

File: docs/semgrep-ci/overview.md [hunk 6]
```diff
@@ -1,262 +1,155 @@
...
-- [GitHub pull request comments](/semgrep-app/notifications/#github-pull-request-comments)
-- GitLab merge request comments ([sign up for the beta here](https://go.r2c.dev/join-gitlab-beta))
-- [Slack messages](/semgrep-app/notifications/#slack)
-- [emails](/semgrep-app/notifications/#email)
-- [webhooks](/semgrep-app/notifications/#webhooks) (paid feature in Semgrep App)
+#### Default behaviors of scans set up through Semgrep App
 
-To set up notifications, see [Notifications document](/semgrep-app/notifications) for more information.
+* Findings do not block a PR or MR. The presence of findings can be customized to block a PR or MR through the [Rule Board](docs/semgrep-app/rule-board).
+* The CI job does not fail when detecting findings, but can still fail due to other errors. See [Semgrep error codes](../cli-reference/#error-codes).
```
Comment (pabloest) [inline docs/semgrep-ci/overview.md:118]: I got a 404 when following this link. I think the one you want is `/docs/cli-reference/#exit-codes`?

File: docs/semgrep-ci/overview.md [hunk 7]
```diff
@@ -1,262 +1,155 @@
...
 
-- [GitHub pull request comments](/semgrep-app/notifications/#github-pull-request-comments)
-- GitLab merge request comments ([sign up for the beta here](https://go.r2c.dev/join-gitlab-beta))
-- [Slack messages](/semgrep-app/notifications/#slack)
-- [emails](/semgrep-app/notifications/#email)
-- [webhooks](/semgrep-app/notifications/#webhooks) (paid feature in Semgrep App)
+#### Default behaviors of scans set up through Semgrep App
 
-To set up notifications, see [Notifications document](/semgrep-app/notifications) for more information.
+* Findings do not block a PR or MR. The presence of findings can be customized to block a PR or MR through the [Rule Board](docs/semgrep-app/rule-board).
```
Comment (pabloest) [inline docs/semgrep-ci/overview.md:117]: I got a 404 when following this link. I think the one you want is `/docs/semgrep-app/rule-board/`.

File: docs/semgrep-ci/overview.md [hunk 8]
```diff
@@ -1,262 +1,155 @@
...
 
-Notifications require connection to Semgrep App. You can get notified about new findings via:
+1. [Sign in](https://semgrep.dev/login) to Semgrep App.
+2. Click on **Projects > Scan New Project**.
+3. Follow instructions in the App. These steps vary based on your CI provider.
+4. Semgrep App generates a `SEMGREP_APP_TOKEN` and creates a configuration file. 
+5. Commit the configuration file and add `SEMGREP_APP_TOKEN` as a credential, token, or secret.
+6. The scan automatically begins.
+7. After the scan, Semgrep App reports findings (if any) into the Findings dashboard.
+8. Optional: If you are not satisfied with your CI job's behavior, troubleshoot and edit the configuration file. Refer to [Sample CI configuration for Semgrep App](semgrep-ci/sample-ci-configs).
```
Comment (pabloest) [inline docs/semgrep-ci/overview.md:112]: I got a 404 when following this link. I think the one you want is `/docs/semgrep-ci/sample-ci-configs/`.

**Review by pabloest — APPROVED** [review]
Pre-approving as I think all the comments are minor.

**Pull request merged**